.github/workflows/fork-ci.yaml

```diff
@@ -0,0 +1,107 @@
+        echo DOCKER_TAG=${{ github.ref_name }} >> $GITHUB_ENV
+    - name: Store docker registry and tag for following jobs
+      id: store-docker-registry-and-tag
+      run: |
+        echo DOCKER_REGISTRY=$DOCKER_REGISTRY >> $GITHUB_OUTPUT
+        echo DOCKER_TAG=$DOCKER_TAG >> $GITHUB_OUTPUT
+    outputs:
+      docker_registry: ${{ steps.store-docker-registry-and-tag.outputs.DOCKER_REGISTRY }}
+      docker_tag: ${{ steps.store-docker-registry-and-tag.outputs.DOCKER_TAG }}
+
+  build_and_push_images:
+    needs: determine_docker_registry_and_tag
+    runs-on: ubuntu-latest
+    env:
+      BUILD_PLATFORMS: linux/amd64,linux/arm64,linux/ppc64le
+      DOCKER_REGISTRY: ${{ needs.determine_docker_registry_and_tag.outputs.docker_registry }}
+      DOCKER_TAG: ${{ needs.determine_docker_registry_and_tag.outputs.docker_tag }}
+    strategy:
+      matrix:
+        include:
+        - component: controller
+          image_name: ${{ github.event.repository.name }}
+          dockerfile: Dockerfile
+        - component: config-daemon
+          image_name: ${{ github.event.repository.name }}-config-daemon
+          dockerfile: Dockerfile.sriov-network-config-daemon
+        - component: webhook
+          image_name: ${{ github.event.repository.name }}-webhook
+          dockerfile: Dockerfile.webhook
+    steps:
+    - uses: actions/checkout@v4
+    - uses: docker/setup-qemu-action@v3
+    - uses: docker/setup-buildx-action@v3
+    - uses: docker/login-action@v3
+      with:
+        registry: ${{ env.DOCKER_REGISTRY }}
+        username: ${{ secrets.NVCR_USERNAME }}
+        password: ${{ secrets.NVCR_TOKEN }}
+    - uses: docker/build-push-action@v4
+      with:
+        platforms: ${{ env.BUILD_PLATFORMS }}
+        context: .
+        file: ${{ matrix.dockerfile }}
+        tags: ${{ env.DOCKER_REGISTRY }}/${{ matrix.image_name }}:${{ env.DOCKER_TAG }}
+        push: true
+
+  update_network_operator_values:
+    needs:
+    - determine_docker_registry_and_tag
+    - build_and_push_images
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_TAG: ${{ needs.determine_docker_registry_and_tag.outputs.docker_tag }}
+      GH_TOKEN: ${{ secrets.GH_TOKEN_NETWORK_OPERATOR }}
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        token: ${{ secrets.GH_TOKEN_NETWORK_OPERATOR }}
+        repository: ${{ github.repository_owner }}/network-operator
+        sparse-checkout-cone-mode: false
+        sparse-checkout: deployment/network-operator/values.yaml
+    - name: Create PR to update image tags in network-operator values
+      run: |
+        git config user.name  nvidia-ci-cd
+        git config user.email svc-cloud-orch-gh@nvidia.com
+
+        git checkout -b feature/update-sriov-tags-to-$DOCKER_TAG
+        sed -Ei "s/:network-operator-.+/:$DOCKER_TAG/g" deployment/network-operator/values.yaml
```

Comment:
Need to update  `hack/release.yaml`  and run `make release-build`

sed 'version: network-operator-' to lookup in `hack/release.yaml` and set new docker tags.
Make sure Go available and run  `make release-build`

Reply:
+1 on @rollandf's comment

Reply:
changed